.circleci/config.yml

```diff
@@ -408,7 +437,8 @@ jobs:
                 -v /tmp/data:/tmp/data:ro \
                 -v /tmp/fslicense:/tmp/fslicense:ro \
                 -v /tmp/ds005:/tmp/ds005 \
-                -v /tmp/nipype.cfg:/home/smriprep/.nipype/nipype.cfg \
+                -v /tmp/templateflow:/home/smriprep/.cache/templateflow \
+                -v /tmp/src/smriprep/.circleci/nipype.cfg:/home/smriprep/.nipype/nipype.cfg \
```

Comment:
Does nipype know to look there?

Reply:
That's $HOME within docker and doesn't make any difference to most singularity installations